public/video-ui/src/services/VideosApi.js

```diff
@@ -258,5 +258,22 @@ export default {
       }
       return '';
     });
+  },
+
+  preventPublication(composerId, composerUrl) {
+    function doEmbargoIndefinitely(stage) {
+      return pandaReqwest({
+        url: `${composerUrl}/api/content/${composerId}/${stage}/settings/embargoedIndefinitely`,
+        method: 'put',
+        crossOrigin: true,
+        withCredentials: true,
+        data: `"true"`
```

Comment:
Alternatively you could change the creation method in composer to embargo if the embargo is set to true in the create request body (which should be sent automatically). But that would require making another (simple) pr.